org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetBridgeHandler.java

```diff
@@ -380,7 +381,40 @@ public void onMessage(OpenMessage msg) {
         }
 
         BaseOpenMessage baseMsg = (BaseOpenMessage) msg;
+        // ATTENTION to this logic IF with future implementation of a bus group commands
+        if (deviceDiscoveryService.isGeneral(baseMsg.getWhere()) && baseMsg instanceof Auxiliary) {
+            // This is to avoid creating the thing via discoveryByActivation
+            // command Auxiliary general
+            logger.debug("==OWN==  Message Auxiliary General");
+            return;
+        }
+        if (deviceDiscoveryService.isGeneral(baseMsg.getWhere()) || deviceDiscoveryService.isArea(baseMsg.getWhere())) {
+            if (baseMsg instanceof Automation) {
+                if (isLocalBus(baseMsg)) {
+                    // command Automation Local bus general or ambient
+                    logger.debug("==OWN==  Message Local bus Automation General or Area");
+                    callAllThingHandlerMessage(msg, Who.AUTOMATION.value().toString());
+                } else {
+                    // This is to avoid creating the thing via discoveryByActivation
+                    // command Automation general or ambient
+                    logger.debug("==OWN==  Message Automation General or Area");
+                }
+                return;
+            }
+            if (baseMsg instanceof Lighting) {
+                // command Lighting general or ambient
+                logger.debug("==OWN==  Message Lighting General or Area");
+                callAllThingHandlerMessage(msg, Who.LIGHTING.value().toString());
+                return;

```

Comment:
**Promemoria:**
Da eliminare `return;`
Successivamente è stato eliminato nel PR `[FIX #35] Support for group addresses (WHERE=#n)`